lib/hutch/worker.rb

```diff
@@ -103,5 +103,10 @@ def error_acknowledgements
     private
 
     attr_accessor :setup_procs
+
+    def consumer_tag(queue)
+      prefix = Hutch::Config[:consumer_tag_prefix]
+      queue.channel.generate_consumer_tag(prefix)
```

Comment:
So, MarchHare does not have the same interface:

https://github.com/ruby-amqp/march_hare/search?utf8=%E2%9C%93&q=generate_consumer_tag